src/common/front-end-routing/routes/cluster/config/leases/leases-route.injectable.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import isAllowedResourceInjectable from "../../../../../utils/is-allowed-resource.injectable";
+import { frontEndRouteInjectionToken } from "../../../../front-end-route-injection-token";
+
+const priorityClassesRouteInjectable = getInjectable({
+  id: "leases",
```

Comment:
1. This variable name probably should be something like "leasesRouteInjectable"
2. This ID (which has to be globally unique) is a bit too generic, could you change it to be "leases-route"?

Reply:
Fixed